File: Java/UberBlack.java
import java.util.*;

class UberBlack extends Car{
    
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial; 

    public UberBlack(String license, Driver driver,
    Map<String, Map<String, Integer>> typeCarAccepted,
    ArrayList<String> seatsMaterial){

        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;

    }


}

File: PHP/uberBlack.php
<?php
require_once('car.php');
class UberBlack extends Car{
    public $typeCarAccepted;
    public $satsMaterial;

    Public function __construct ($license, $driver, $typeCarAccepted, $satsMaterial){
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->satsMaterial = $satsMaterial;
        }
}
?>